### ISSUE TITLE
"SetLanePosition" in esminiRMLib according to LHT maps incorrect

### ISSUE THREAD

**Issue by MarcHerrmannTT** [esmini/esmini#581]
We have an issue according to the RMLibs "SetLanePosition" applied on roads using left hand traffic.

**Our usecase:** 
We want realign an object to a given roadnetwork only knowing its world position.  Therefore we use SetLanePosition to get back the correct object heading according to the relevant lanes driving direction.
This works fine for RHT maps but on LHT maps the object always looks in opposite direction.

Roughly we use the following approach using the c# wrapper:

```csharp
SetWorldPosition(id, x, z, y, h, p, r);
GetPositionData(id, ref positionData);
SetLanePosition(id, positionData.roadId, positionData.laneId, positionData.laneOffset, positionData.s, true);
GetPositionData(id, ref positionData);
return positionData.h;
```

I would assume that here within the function the maps LHT rule must be taken into account as well:
https://github.com/esmini/esmini/blob/41c36c98f1e86b3534657b11ab0d23734a6387b4/EnvironmentSimulator/Libraries/esminiRMLib/esminiRMLib.cpp#L428C1-L440C31

Best regards,
 
Marc

**Comment by eknabevcc** [comment]
Thanks for reporting this flaw. Fix included in [v2.37.14](https://github.com/esmini/esmini/releases/tag/v2.37.14).

Please try and report back if it works as expected or not.

**Comment by MarcHerrmannTT** [comment]
Good morning, Emil.
At first glance that looks good. Thanks for the very fast fix!
Have a nice day
